src/mods.cpp

```diff
@@ -29,14 +29,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "convert_json.h"
 #include "exceptions.h"
 
-static bool parseDependsLine(std::istream &is,
-		std::string &dep, std::set<char> &symbols)
+static bool parseDependsString(std::string &dep,
```

Comment:
this function looks like it should be called `trimDependsString` for clarity. It trims the string by parsing it.